src/experimental/patterns/ec2-app.ts

```diff
@@ -324,15 +341,15 @@ export interface GuUserDataForRollingUpdateExperimentalProps {
 export class GuUserDataForRollingUpdateExperimental {
   constructor(scope: GuStack, props: GuUserDataForRollingUpdateExperimentalProps) {
```

Comment:
And we'll also need to pass the slow start duration here if we enable this for MAPI.